help/en/html/tools/Audio.shtml

```diff
@@ -30,16 +30,18 @@ they currently contain.<br>
 <a href="../../package/jmri/jmrit/beantable/AudioTable.jpg">
 <img src="../../package/jmri/jmrit/beantable/AudioTable.jpg" height="178" width="420">
 </a>
-<p>From the Audio Table, you can 
-<a href="../../package/jmri/jmrit/beantable/AudioAddEdit.shtml">create a new Audio object</a>.
+
+<h4>Audio Table controls</h4>
+<p>Below the Audio Table are two buttons, named "Add Source..." and "Add Buffer..." to 
+<a href="../../package/jmri/jmrit/beantable/AudioAddEdit.shtml">create new Audio object</a>.
```

Comment:
No problem @mattharris and thanks for scanning the text. I'll fix that one asap.